### ISSUE TITLE
support for custom selectors

### ISSUE THREAD

**Issue by shanebdavis** [generalui/hooks-for-redux#9]
I'd like to add the ability to optionally select a subset of the H4R redux slice and only re-render when that subset changes:

```javascript
const [use, {...}] = createReduxModule('example', {foo: 1, bar: 2}, {...})

const MyComponent = () => {
  const foo = use(({foo}) => foo) // doesn't re-render if bar changes
  return ...
}
```

**Comment by mattcgenui** [comment]
This should take care of it: #11

**Comment by shanebdavis** [comment]
This is now in

**Comment by shanebdavis** [comment]
@mattcgenui Thanks for this! I finally got this released as version 2.1. I want to add typescript support for this too, so I created a new issue for tracking that effort.
